Release notes for milestone 8.2 (#117)
NethServer/ns8-docs: software_center.rst

- *Core* contains the web user interface, the API server, the agents
  that manage the containers and backup engines.
- *LDAP proxy* is a proxy server for LDAP TCP connections that handles all
  connections from applications to :ref:`User domain providers
  <user-domains-section>`.
- :ref:`Loki log server <loki-section>`
- :ref:`Traefik proxy <traefik-section>`
- :ref:`Samba Active Directory <active_directory-section>`
- :ref:`OpenLDAP <openldap-section>`

You can review the components currently installed in the cluster at any
time. To do this, click on the three-dots menu in the top-right corner of
the ``Software Center`` page, then select the ``Core apps`` option.
Click on :guilabel:`Update core` button to apply the updates.

Core module updates are always applied altogether to avoid version
mismatches.

.. _module_updates-section:

.. _application-updates:

Application updates
-------------------

The list of available updates is listed inside the ``Updates`` tab of
:ref:`software_center-section`. The software center displays a card for
each application with available updates.

You can apply all application updates by clicking the :guilabel:`Update all
apps` button.

> By clicking the ``Update details`` link on the application card, you will

Reviewer: There is no `Update details` link in the UI :thinking:

see all module instances that require an update. You can update each
instance separately by clicking on the :guilabel:`Update` button. If you
prefer to update all instances of the same module, just click
:guilabel:`Update all instances` button.

.. _move_clone-section:

Clone and move applications
===========================

> An application instance can be cloned inside a cluster node. The clone

Reviewer: ```suggestion
An application instance can be cloned to any cluster node. The clone
```
